.github/workflows/main.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         activate-environment: ${{ env.CONDA_ENV }}
         python-version: 3.8
         # Use miniforge to only get conda-forge as default channel.
-        miniforge-version: latest
+        # miniforge-version: latest
```

Comment:
@tomMoral removing this line breaks the build. So it confirms we do not add conda-forge to the channels through the env.yml as we thought we did. I am fixing it, will send a PR